app/Model/Event.php

```
@@ -6372,12 +6372,11 @@ public function enrichment($params)
                             throw new MethodNotAllowedException(h($module['name']) . ' service not reachable.');
                         } else if (!is_array($result)) {
                             continue 2;
+                        } else if (!isset($result['results'])) {
+                            throw new RuntimeException("Invalid response received from module {$module['name']}, response data do not contains results field.");
```

The `Enrich Event` functionally now is breaking when exception is returned by this line.
Earlier, the `Enrich Event` ignores errors and complete on processing the reminder of attributes and objects.
When you look at number of the misp-modules, its expected to return`misperrors['error']` for attributes that were unable to be enriched (e.g. [revresedns.py](https://github.com/MISP/misp-modules/blob/main/misp_modules/modules/expansion/reversedns.py)).